ScintSuite/LibData/AUG/Misc.py

```diff
@@ -242,6 +242,25 @@ def get_icrh(shot: int, coupled: bool = False):
     output.attrs['signal'] = name
     return output
 
+
+def get_ecrh(shot: int, coupled: bool = False):
```

Comment:
get_ECRH_traces and get_ECRH_power from the equilibrium library make this, just add a xArrayOutput flag if you want them in xarray, no need of a new function